src/zope/app/apidoc/utilities.rst

```diff
@@ -166,13 +166,15 @@ For interfaces we simply get
   >>> utilities.getPythonPath(ISample)
   'zope.app.apidoc.doctest.ISample'
 
-and for classes
+and for classes (note that we must pass a "bound method", because the
+unbound method is no longer present in Python 2):
 
   >>> class Sample(object):
   ...     def sample(self):
   ...         pass
 
-  >>> utilities.getPythonPath(Sample.sample)
+  >>> PY3 = str is not bytes
+  >>> utilities.getPythonPath(Sample().sample if PY3 else Sample.sample)
   'zope.app.apidoc.doctest.Sample'
```

Comment:
The problem is that they both produce exactly the same under Python 2:
```python
  >>> utilities.getPythonPath(Sample().sample)
  'zope.app.apidoc.doctest.Sample'
  >>> utilities.getPythonPath(Sample.sample)
  'zope.app.apidoc.doctest.Sample'
```

But they produce different things under Python 3:
```python
  >>> utilities.getPythonPath(Sample().sample)
  'zope.app.apidoc.doctest.Sample'
  >>> utilities.getPythonPath(Sample.sample)
  'zope.app.apidoc.doctest.sample' 
  # note the lower case 's', because it's just a function, 
  # not an 'unbound method', which no longer exists
```

That's annoyingly ugly to try to handle in doctests. I'm open to suggestions, though!